fedimint-core/src/fee_consensus.rs

```diff
@@ -19,7 +19,7 @@ impl FeeConsensus {
     /// # Errors
     /// - This constructor returns an error if the relative fee is in excess of
     ///   one thousand parts per million.
-    pub fn new(parts_per_million: u64) -> anyhow::Result<Self> {
+    pub fn new_lnv2(parts_per_million: u64) -> anyhow::Result<Self> {
```

Comment:
I am **very** opposed to having any module-specific code in `fedimint-core`.

Reply:
I agree. I am not too happy with how this has turned out either. I would keep it like that for now - most important thing here is to make sure that the consensus fee calculation is correct. Eventually I want to refactor this though.